lib/tasks/taxonomy_event_log.rb

```diff
@@ -7,9 +7,9 @@ def export
       previous_taxons = previous_events[event.content_id].to_a
       current_taxons = event.payload[:links][:taxons]
```

Comment:
Might want to `event.payload.fetch(:links, {})[:taxons]` in case there's no `link` hash